lib/ynab/models/user_response_data.rb

```diff
@@ -100,37 +100,30 @@ def hash
     # @param [Hash] attributes Model attributes in the form of hash
     # @return [Object] Returns the model itself
     def self.build_from_hash(attributes)
-      new.build_from_hash(attributes)
```

Comment:
Are all these changes coming directly from the generator?  Were any of them made manually?